users/views.py

```diff
@@ -10,3 +12,22 @@ class SignUpView(CreateView):
     form_class = SignUpForm
     template_name = 'users/signup.html'
     success_url = '/'
+
+
+class FavedSeriesView(LoginRequiredMixin, ListView):
+    template_name = "users/followed_series.html"
+    context_object_name = 'faved_series_list'
+
+    login_url = '/accounts/login/'
```

Comment:
Is this necessary? Django should be using the `LOGIN_URL` which we already defined in `settings.py`.